api/circulation.py

```diff
@@ -77,7 +94,16 @@ def __repr__(self):
 
 class HoldInfo(CirculationInfo):
 
-    """A record of a hold."""
+    """A record of a hold.
+
+    :param identifier_type Ex.: Identifier.ONECLICK_ID.
+    :param identifier Expected to be the unicode string of the isbn, etc..
+    :param start_date When the patron made the reservation.
+    :param end_date When checked-out book is due.  Expected to be passed in 
+        date, not unicode format.
```

Comment:
The end_date of a hold is the date the book is estimated to be available to this patron.